server/src/Ogmios/Parser.hs

```diff
@@ -0,0 +1,146 @@
+module Ogmios.Parser (decodeProtocolParameters) where
```

Comment:
This applies to most of this module: since you're just using `Text` as the error type, why bother with `Data.Aeson.BetterErrors`? We just use Aeson directly everywhere else and now we need to depend on parsec as well

Reply:
Those are two separate things coming from the same problem: 

- The original parser was written in pure Aeson but it was crashing silently, so, I introduced `BetterErrors` to get some feedback for debugging. 
- There were various problems, in particular, Ogmios returns `Rationals` as strings `"a/b"` and the `FromJSON` instance for `Rational` didn't expect this. That's the reason we now import `Parsec`, so we can parse these rationals.

At first, I thought of writing the `Rational` parser ad-hoc but that would mean writing part of the core of `Parsec` in the module anyways. 

I can come back to pure Aeson with some ease once I solve the `how to parse praosNonce` issue.

Reply:
The problem with `extra entropy` is solved now, so, I can convert the parser back to pure Aeson, it would take me around 4 hours, should I do it?

Reply:
I would prefer using Aeson directly, for a few reasons:
- there's no extra dependencies
- possible contributors are less likely to know Parsec and aeson-better-errors, while we can assume that they will be comfortable using Aeson parsers
- we can remove the custom Parsec parsers
- Aeson uses attoparsec, which is more efficient than parsec

Reply:
That said, if you really don't want to rewrite the parser or if it will take too much time, you can leave it as-is

Reply:
Converted to Aeson. 

Fixed.